extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_logo.php

```diff
@@ -0,0 +1,18 @@
+<? if ( !isset( $model['header']['subtitle'] ) ): ?>
+<a href="<?= Sanitizer::encodeAttribute( $model['header']['href'] ); ?>" class="wds-global-navigation__logo">
+	<?= DesignSystemHelper::getSvg(
+		$model['header']['image'],
+		'wds-global-navigation__logo-fandom'
+	) ?>
+</a>
+<? else: ?>
+<a href="<?= Sanitizer::encodeAttribute( $model['header']['href'] ); ?>" class="wds-global-navigation__logo wds-global-navigation__logo-wikia">
```

Comment:
How about `wds-global-navigation__logo-with-subtitle`? And remove `fandom` from the standard one? That way we don't care which logo it is.